presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -389,6 +397,24 @@ private synchronized PrioritizedSplitRunner pollNextSplitWorker()
         return null;
     }
 
+    private void monitorActiveSplits()
+    {
+        Iterator<RunningSplitInfo> iterator = runningSplitInfos.iterator();
```

Comment:
Thanks for pointing this out! Given we are going to print out every split's call stack, will use while loop instead